src/main/java/de/tum/in/www1/artemis/web/rest/TextExerciseResource.java

```diff
@@ -87,6 +97,11 @@ public TextExerciseResource(TextExerciseRepository textExerciseRepository,
             !authCheckService.isAdmin()) {
             return forbidden();
         }
+
+        if (textExercise.getAssessmentDueDate() == null) {
+            textExercise.setAssessmentDueDate(textExercise.getDueDate().plusWeeks(1));
```

Comment:
What happens if dueData is null? Please add another null check

Reply:
Done